jobs/shield-daemon/spec

```diff
@@ -9,68 +9,83 @@ templates:
   bin/monit_debugger:   bin/monit_debugger
   config/shieldd.conf:  config/shieldd.conf
   shared/id_rsa:        shared/id_rsa
-  data/properties.sh:   data/properties.sh
   helpers/ctl_setup.sh: helpers/ctl_setup.sh
   helpers/ctl_utils.sh: helpers/ctl_utils.sh
 
+provides:
+- name: shield-daemon
+  type: shield-daemon
+  properties:
+  - domain
+  - port
+  - http_port
+  - auth.api_keys
+
+consumes:
+- name: shield-db
+  type: sql
+- name: sessions-db
```

Comment:
Fixed, thanks. And the rest of the templates support explicit `p('database.host')` declaration of database.